### ISSUE TITLE
[Feature Request] Add Flat Hit Points Bonus Advancement

### ISSUE THREAD

**Issue by Zanderaf** [foundryvtt/dnd5e#2599]
A few races / subclasses give out a flat amount of maximum hit points per level, like a partial Tough feat. Is there any chance something like this could be added to advancement? I know there's the option of adjusting it with a per level bonus via the hp tab in the sheet, but it would be nice it was an option as well. Two main places I remember it being in is Hill Dwarves & Draconic Bloodline Sorcerers

**Dwarven Toughness.** Your hit point maximum increases by 1, and it increases by 1 every time you gain a level.
**Draconic Resilience.** At 1st level, your hit point maximum increases by 1 and increases by 1 again whenever you gain a level in this class.

**Comment by Ikaguia** [comment]
This can be currently be done via an active effect adding `+1` to `system.attributes.hp.bonuses.level on the relevant feature.

**Comment by Zanderaf** [comment]
> This can be currently be done via an active effect adding `+1` to `system.attributes.hp.bonuses.level` on the relevant feature.

True, was just meaning a separate, non AE / Manual edited way of doing it. Been using an AE but sometimes they get weird for whatever reason

**Comment by Zanderaf** [comment]
Any chance this might be at all worked on for 3.1.0?
